README.md

```diff
@@ -96,6 +96,7 @@ Additional lists you might find useful:
 ## Auditing / Logging
 *Plugins for auditing and logging.*
 
+- [Activity Logger plugin](https://github.com/elstc/cakephp-activity-logger) - ActivityLogger plugin for CakePHP.
```

Comment:
The description could explain a bit more, and the "plugin for CakePHP" can be dropped as this is obvious.